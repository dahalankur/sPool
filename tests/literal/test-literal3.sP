# bool
true
false
True
False
TRUE
FALSE
