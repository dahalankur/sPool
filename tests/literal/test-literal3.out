PROGRAM {[ 
    BOOL(true) 
    BOOL(false) 
    VAR(True) 
    VAR(False) 
    VAR(TRUE) 
    VAR(FALSE)]}
